tests/PasswordCheckerTest.php

```diff
@@ -1,10 +1,10 @@
 <?php
-use FusionsPim\PhpPasswordChecker\PasswordChecker;
+use FusionsPim\PhpPasswordChecker\{PasswordChecker, PasswordException};
 use PHPUnit\Framework\TestCase;
 
 class PasswordCheckerTest extends TestCase
 {
-    protected function setUp()
+    protected function setUp(): void
```

Comment:
If this has `:void` return type, should all the tests have it too?

Reply:
@WillJW I think the tests are fine as they are, however `setUp()` needs the return type for the `phpunit` update.